lib/jobs/uefi-job.js

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2016, EMC, Inc.
```

Comment:
@yyscamper didn't find any file format problems in on-tasks. I had to fix format in on-taskgraph but I am still seeing the issue